pages/common/git-commit.md

```diff
@@ -22,3 +22,7 @@
 - Commit only specific (already staged) files:
 
 `git commit {{path/to/file1}} {{path/to/file2}}`
+
+- Commit staged files to the repository with a message while allowing the commit be empty:
```

Comment:
How about just
```suggestion
- Allow empty commits:
```

Reply:
I don't think so. It doesn't describe it and doesn't fit with the page.

```suggestion
- Create a commit which has the ability to be empty with a message
```

Reply:
That's too wordy.  How about
```suggestion
- Create an empty commit with no added, deleted, or changed files:
```

Reply:
It isn't mandatory for it to empty, the flag just allows it to be.

Reply:
If there are staged commits, it will put them in.

Reply:
Another suggestion:
```suggestion
- Create a commit, even if there are no staged files:
```

Reply:
Yes thats fine

Reply:
....but surely you wouldn't use this unless you wanted an empty commit in the first place? Either of @marchersimon's suggestions would be fine.

Reply:
You might use it in a script thats automated.